### DISCUSSION TITLE
[Question][Hw1] Question about passing vigenere crack test on autograde

### DISCUSSION THREAD

**Discussion by ahnjonggun** [prosyslab-classroom/cs348-information-security#670] [Q&A]
Name: Jaewoo Ahn

I have two versions of my code on vigenere crack. One of them does not seem to "pass" the autograder test and is showed as failed(red) in the result. However, its total score is higher than the one which passes the vigenere crack test(the tests all show up green).
I wonder what exactly passing a test(vigenere test especially) means, and if it is okay to have some of my tests failed.
If a test fails, will any score which was given by the failed test be ignored? Or pass/fail of individual tests don't matter?

Thank you.

**Comment by yeonhee-ryou** [comment]
"pass" means that all of your answers got score greater than zero, and
"fail" means that at least one of your answer got zero point.

**Comment by ahnjonggun** [comment]
> "pass" means that all of your answers got score greater than zero, and "fail" means that at least one of your answer got zero point.

Thank you!
